kansha/board/comp.py

```diff
@@ -325,12 +324,8 @@ def update_card_position(self, data):
             dest = cols[data['dest']]
             card = orig.remove_card(data['card'])
             dest.insert_card(card, data['index'])
-            values = {'from': orig.title().text,
-                      'to': dest.title().text,
-                      'card': card().data.title}
-            notifications.add_history(self.data, card().data,
-                                      security.get_user().data,
-                                      u'card_move', values)
+            event = ('card_move', [card(), dest], orig)
```

Comment:
Card is a business object here, it has no answer method. 
The question is indeed: can an upper level explicitly emit an event on lower level components? Without that lower component even knowing?